client/my-sites/checklist/wpcom-checklist/index.js

```diff
@@ -288,6 +362,8 @@ export default connect(
 			siteSlug,
 			taskStatuses: get( getSiteChecklist( state, siteId ), [ 'tasks' ] ),
 			taskUrls,
+			userEmail: user && user.email,
```

Comment:
In terms of code, it might be possible for `userEmail` to be `false` or `undefined` here, which could leave us showing:

"Please click the link in the email we sent at {false/undefined}."

I'm not sure what the best solution would be, maybe only render the component if there's an email? Maybe show some generic or alternative messaging?

Feel free to ignore though if this is a non-issue, as in, we can rely on email being present

Reply:
+1 -- we may see a flash of `false` or `undefined`.

Let's just make it an empty string if it's not set...

`userEmail: ( user && user.email ) || '',`